lib/commands/context.js

```diff
@@ -117,12 +122,27 @@ commands.mobileGetContexts = async function mobileGetContexts () {
   return await webviewHelpers.getWebViewsMapping(this.adb, opts);
 };
 
-helpers.switchContext = async function switchContext (name) {
+helpers.getContextsWithWebviewMappings = function getContexts (webViewsMapping) {
+  let webviews;
+  if (this.isChromeSession) {
+    // if we have a Chrome browser session, we only care about the Chrome
+    // context and the native context
+    webviews = [CHROMIUM_WIN];
+  } else {
+    // otherwise we use ADB to figure out which webviews are available
+    webviews = webviewHelpers.getWebviews(webViewsMapping, this.opts.ensureWebviewsHavePages);
+  }
+  this.contexts = _.union([NATIVE_WIN], webviews);
```

Comment:
you could also use `this.contexts = [NATIVE_WIN, ...webviews]`

Reply:
Addressed in a83c7b2